### DISCUSSION TITLE
Installing/Removing features after deployment

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#7433] [Unsupported Versions] [answered]
In this case, I deployed a platform without the Wazuh component, which I am now interesting in adding to the ManagerSearch Node. How would I go about adding Wazuh and Ossec to the Management Node and pushing the agents to the existing nodes etc? Thanks.

**Comment by dougburks** [comment, marked as answer]
The supported method of enabling Wazuh is at install time. You may be able to enable it afterwards, but this is untested and unsupported.
